ansible_runner/__main__.py

```diff
@@ -756,6 +762,17 @@ def main(sys_args=None):
     vargs = vars(args)
 
     if vargs.get('command') in ('worker', 'process'):
+        if vargs.get('worker_info'):
+            cpu = get_cpu_capacity()
+            mem = get_mem_capacity()
+            if cpu or mem > 0:
+                print("\nExecution Node Info\n"
```

Comment:
How are you planning to parse this data? It might be easier to just make this print json.